tests/Support/JsonApi/V1/Sites/Capabilities/CrudSite.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace LaravelJsonApi\OpenApiSpec\Tests\Support\JsonApi\V1\Sites\Capabilities;
+
+use Illuminate\Support\Str;
+use LaravelJsonApi\NonEloquent\Capabilities\CrudResource;
+use LaravelJsonApi\OpenApiSpec\Tests\Support\Entities\Site;
+use LaravelJsonApi\OpenApiSpec\Tests\Support\Entities\SiteStorage;
+
+class CrudSite extends CrudResource
+{
+    protected SiteStorage $sites;
+
+    public function __construct(SiteStorage $sites)
+    {
+        parent::__construct();
+
+        $this->sites = $sites;
+    }
+
+    public function create(array $validatedData): Site
+    {
+        $site = new Site($validatedData['slug']);
+        $site->setDomain($validatedData['domain'] ?? null);
+        $site->setName($validatedData['name'] ?? null);
+
+        $this->sites->store($site);
+
+        return $site;
+    }
+
+    public function read(Site $site): ?Site
+    {
+        $filters = $this->queryParameters->filter();
+
+        if ($filters && $name = $filters->value('name')) {
```

Comment:
A name filter isn't registered, is that expected?

Reply:
It does not need to be registered in order to work due to the custom Capabilities, but its better for generation of the document to actually add it to the schema, otherwise you won't get the filter in the documentation.
So I added it to the schema filters.